vmm/src/seccomp_filters.rs

```diff
@@ -528,6 +528,8 @@ fn vmm_thread_rules() -> Result<Vec<(i64, Vec<SeccompRule>)>, BackendError> {
         (libc::SYS_unlink, vec![]),
         #[cfg(target_arch = "aarch64")]
         (libc::SYS_unlinkat, vec![]),
+        #[cfg(target_arch = "aarch64")]
```

Comment:
In your commit message, please use "authorized list" instead of "whitelist".

Reply:
Thanks @sboeuf.